db/fixtures/miq_shortcuts.yml

```diff
@@ -26,7 +26,7 @@
   :startup: true
 - :name: chargeback_assignments
   :description: Overview / Chargeback / Assignments
-  :url: /chargeback_assignments/explorer
+  :url: /chargeback_assignment
```

Comment:
Wait, why are we going to singular here?

Reply:
plural was a typo, it should have been a singular.